test/system/test_cvp_client_api.py

```diff
@@ -585,16 +457,16 @@ def test_api_get_configlet_builder(self):
         # Verify the following keys and types are
         # returned by the request
         exp_data = {
-            u'isAssigned': bool,
-            u'formList': list,
-            u'main_script': dict,
+            'isAssigned': bool,
+            'formList': list,
+            'main_script': dict,
         }
         # Handle unicode type for Python 2 vs Python 3
         if sys.version_info.major < 3:
-            exp_data[u'name'] = (unicode, str)
+            exp_data['name'] = str
```

Comment:
These unicode formats were in reference to a difference between Py2 and Py3. Is the none unicode format standard for Python3? Will this deprecate Py2?

Reply:
Reverted this change.